README.md

```diff
@@ -122,7 +122,7 @@ The ClearML Agent executes experiments using the following process:
 
 #### System Design & Flow
 
-<img src="https://allegro.ai/clearml/docs/_images/ClearML_Architecture.png" width="100%" alt="clearml-architecture">
+<img src="https://github.com/allegroai/clearml-agent/blob/master/docs/clearml_architecture.png" width="100%" alt="clearml-architecture">
```

Comment:
@pollfly the file referenced by `https://github.com/allegroai/clearml-agent/blob/master/docs/clearml_architecture.png` does not exist in the repository

Reply:
@jkhenning I added the image file in commit [27cf7dd](https://github.com/allegroai/clearml-agent/blob/b76e4fc02b22c09d216e17973d07cefd2147fbff/docs/clearml_architecture.png)

Reply:
Sorry, missed that :)